app/src/components/sections/SherTokenView/DividendsBalancesSection.vue

```diff
@@ -40,16 +40,15 @@ import type { Address } from 'viem'
 import CardComponent from '@/components/CardComponent.vue'
 import TableComponent from '@/components/TableComponent.vue'
 import { useContractBalance } from '@/composables/useContractBalance'
+import { useTeamStore } from '@/stores/teamStore'
 import EthereumIcon from '@/assets/Ethereum.png'
 import USDCIcon from '@/assets/usdc.png'
 import MaticIcon from '@/assets/matic-logo.png'
+const teamStore = useTeamStore()
 
-interface Props {
-  address: Address
-}
-const props = defineProps<Props>()
-
-const { dividends, isLoading } = useContractBalance(props.address)
+const { dividends, isLoading } = useContractBalance(
+  teamStore.getContractAddressByType('Bank') as Address
+)
```

Comment:
If you have to ass Address that mean there is a problem. 


getContractAddressByType return Address | Undefine

So useContractBalance should be able to recieve the same thing, and do the opperation only when the address is not undefine.